voicevox_engine/utility/connect_base64_waves.py

```diff
@@ -1,46 +1,50 @@
 import base64
 import io
+import operator
 from typing import List, Tuple
 
 import numpy as np
 import soundfile
+from scipy.signal import resample
 
 
 class ConnectBase64WavesException(Exception):
     def __init__(self, message: str):
         self.message = message
 
 
-def decode_base64_waves(waves: List[str]) -> Tuple[List[np.ndarray], float]:
+def decode_base64_waves(waves: List[str]) -> List[Tuple[np.ndarray, float]]:
```

Comment:
`soundfle.read()`で出力されるサンプリングレートは`int`のはずですが何故か`float`になっていました。

https://python-soundfile.readthedocs.io/en/0.10.3post1/#soundfile.read

Reply:
たぶんintで大丈夫だと思うので、直して頂けると！
```suggestion
def decode_base64_waves(waves: List[str]) -> List[Tuple[np.ndarray, int]]:
```

Reply:
こちら修正しました